Bio/Phylo/TreeConstruction.py

```diff
@@ -374,38 +374,38 @@ class DistanceCalculator(object):
     --------
     >>> from Bio.Phylo.TreeConstruction import DistanceCalculator
     >>> from Bio import AlignIO
-    >>> aln = AlignIO.read(open('Tests/TreeConstruction/msa.phy'), 'phylip')
+    >>> aln = AlignIO.read(open('./TreeConstruction/msa.phy'), 'phylip')
     >>> print(aln)
     SingleLetterAlphabet() alignment with 5 rows and 13 columns
     AACGTGGCCACAT Alpha
     AAGGTCGCCACAC Beta
+    CAGTTCGCCACAA Gamma
```

Comment:
How strange that the original text was out of order.

Reply:
I think the original ordering matched the plotted tree's taxa. (Not that that's a strict ordering either.)